templates/project-ts/package.json

```diff
@@ -36,6 +36,7 @@
     "typescript": "^4.7.2"
   },
   "peerDependencies": {
-    "o1js": "0.15.*"
+    "o1js": "^0.15.4",
```

Comment:
```suggestion
    "o1js": "0.15.*",
```

Reply:
Is there a reason you you changed this? We added the pattern  `"o1js": "^0.15.*"` so we only have to update the cli and push a new version for every minor release of o1js and not every patch release as well.

Reply:
For some reason existing pattern did not used latest o1js/mina-signer, so I decided to bump it like this keeping in mind to revert it back once minor versions published.

Reply:
Maybe it was local cache or something, I'll try with previous pattern again.

Reply:
It would be preferable to revert to the previous pattern if possible so we don't have to release a new cli version when o1js gets bumped to `0.15.5`.